content/kubermatic/main/architecture/compatibility/OS-support-matrix/_index.en.md

```diff
@@ -20,18 +20,18 @@ The following operating systems are currently supported by Kubermatic:
 
 This table shows the combinations of operating systems and cloud providers that KKP supports:
 
-|   | Ubuntu | CentOS | Flatcar | RHEL | Amazon Linux 2 | Rocky Linux |
-|---|---|---|---|---|---|---|---|
-| AWS | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
-| Azure | ✓ | ✓ | ✓ | ✓ | x | ✓ |
-| Digitalocean  | ✓ | ✓ | x | x | x | ✓ |
-| Equinix Metal | ✓ | ✓ | ✓ | x | x | ✓ |
+|                       | Ubuntu | CentOS | Flatcar | RHEL | Amazon Linux 2 | Rocky Linux |
+|-----------------------|--------|--------|---------|------|----------------|-------------|
+| AWS                   | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+| Azure                 | ✓ | ✓ | ✓ | ✓ | x | ✓ |
+| Digitalocean          | ✓ | ✓ | x | x | x | ✓ |
+| Equinix Metal         | ✓ | ✓ | ✓ | x | x | ✓ |
 | Google Cloud Platform | ✓ | x | x | x | x | x |
-| Hetzner | ✓ | ✓ | x | x | x | ✓ |
-| KubeVirt | ✓ | ✓ | ✓ | ✓ | x | ✓ |
-| Nutanix | ✓ | ✓ | x | x | x | x |
-| Openstack | ✓ | ✓ | ✓ | ✓ | x | ✓ |
+| Hetzner               | ✓ | x | x | x | x | ✓ |
```

Comment:
Can we give this "X" a [footnote](https://www.markdownguide.org/extended-syntax/#footnotes) that "Hetzner has deprecated CentOS 7 and will stop supporting it on <date of EOL>"?